packages/schematics/angular/e2e/files/src/app.e2e-spec.ts

```diff
@@ -11,4 +12,13 @@ describe('workspace-project App', () => {
     page.navigateTo();
     expect(page.getTitleText()).toEqual('Welcome to <%= relatedAppName %>!');
   });
+
+  afterEach(async () => {
+    // Assert that there are no errors emitted from the browser
+    const logs = await browser.manage().logs().get('browser');
+    const errors = logs.filter((entry: logging.Entry) => {
```

Comment:
Is using `some()` easier to read here?

```typescript
expect(logs.some((entry: logging.entry) => entry.level === logging.level.SEVERE)).toBe(false);
```

Reply:
Or go fancy (better reading but more obscure API parts):

```typescript
expect(logs).not.toContain(jasmine.objectContaining({ level: logging.level.SEVERE }));
```

Reply:
I prefer the second approach because then jasmine would print the actual logs when the assertion fails.
The first approach would just print `expect  true to be false`, which is not helpful.